src/pages/home/report/ReportActionCompose.js

```diff
@@ -268,6 +269,16 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Get timezone's UTC offset in minutes
+     *
+     * @param {String} [timezone]
+     * @return {Number}
+     */
+    getTimezoneOffset(timezone) {
```

Comment:
Do we need this or can we just call the moment method directly below?

Reply:
We can do it directly below. We can also move this function to date utils if you don't want to pollute the methods of the class.

Reply:
I just don't think we need the method in general tbh. It's just calling another method that already exists

Reply:
Ok, I will delete that method and do the moment stuff directly.

Reply:
Done